styles/stylesheet.css

```diff
@@ -0,0 +1,289 @@
+* {
+  box-sizing: border-box;
+}
+
+/*code for hidden elements*/
+.hidden {
+  position: relative;
+  left: -10000px;
+  top: auto;
+} /*hides the elements off the page*/
+
+.hidden:focus {
+  left: 0;
+}/*makes visible when focused*/
+
+.sr-only {
+  position: absolute;
+  left: -10000px;
+  top: auto;
+  width: 1px;
+  height: 1px;
+  overflow: hidden;
+}
+
+/**HEADER**/
+header {
+  background-color: #083b5f; /*dark blue*/
+  padding: 1% 2% 1% 2%;
+}
+
+header .logo {
+  width: 70%;
+  max-width: 250px;
+  height: auto;
+  margin: 30px 0 0 45px;
+}
+
+.headline {
+  color: #3ca721; /*green color*/
+}
+
+.head-bucket {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+/**Button**/
+.button:link,.button:visited {
+  background-color: #3ca721;
+  color: white;
```

Comment:
Be consistent with colors,if you're using hex values elsewhere, you might as well use them all over. Instead of white, you should use #FFFFFF